kpm/README.adoc

```diff
@@ -266,3 +266,17 @@ kpm install_java_plugin acme:plugin \
 ....
 
 Replace `TOKEN` with a GitHub personal access token.
+
+[[cloudsmith]]
+### Cloudsmith
+
+Starting with version 0.10.0, experimental support for Cloudsmith has been added:
+
+....
+kpm pull_kb_server_war 0.22.21-SNAPSHOT \
+        --overrides url:https://dl.cloudsmith.io/<TOKEN>/<ORG>/<REPO>/maven \
+        --group-id org.kill-bill.billing \
+        --artifact-id killbill-profiles-killbill
```

Comment:
Probably an issue on my end when I uploaded the `war` because in order to make it work i need to use `artifact-id` = `killbill`